e2e/conftest.py

```diff
@@ -0,0 +1,81 @@
+import subprocess
+import time
+from dataclasses import dataclass
+
+import pytest
+from django.contrib.auth import get_user_model
+from scanners.models import ScannerImage
+from selenium import webdriver
+from selenium.webdriver.chrome.options import Options
+from selenium.webdriver.chrome.webdriver import WebDriver
+
+
+@dataclass
+class Config:
+    username: str
+    password: str
+    driver: WebDriver
+
+
+def setup_config(username: str, password: str) -> Config:
+    opts = Options()
+    opts.add_argument("--incognito")
+    opts.add_argument("--headless")
+    return Config(
+        username=username, password=password, driver=webdriver.Chrome(options=opts)
+    )
+
+
+def prep_users(username: str, password: str):
+    User = get_user_model()
+    User.objects.create_superuser(username=username, password=password)
+
+
+def cleanup_users(username: str):
+    User = get_user_model()
+    User.objects.filter(username=username).delete()
+
+
+@pytest.fixture
+def prep_dkron():
+    p = subprocess.Popen(["surface/manage.py", "run_dkron", "-s"])
+    time.sleep(5)  # puke, but required to start dkron properly
```

Comment:
I don't like these two points but I'm not sure there is a better way. Without having to write too much code, I'd like to be able to call this command.
I tried using `call_command` but finding a way to successfully put the process in the background was impossible (using `Process` from `multiprocessing`, `Thread` from `threading`, etc.)